src/components/Home/Home.js

```diff
@@ -0,0 +1,158 @@
+import React ,{ useEffect, useState } from 'react';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Link from '@material-ui/core/Link';
+import img from '../images/nike.jpg'
+import axios from 'axios'
+
+import { withStyles } from "@material-ui/core/styles";
+import  Tab  from './Tab';
+import Item from '../Item/Item'
+
+const useStyles = makeStyles((theme) => ({
+  icon: {
+    marginRight: theme.spacing(2),
+  },
+  heroContent: {
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(8, 0, 6),
+    backgroundImage: `url(${img})`
+  },
+  heroButtons: {
+    marginTop: theme.spacing(4),
+  },
+  cardGrid: {
+    paddingTop: theme.spacing(8),
+    paddingBottom: theme.spacing(8),
+  },
+  card: {
+    height: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  cardMedia: {
+    paddingTop: '56.25%', // 16:9
+  },
+  cardContent: {
+    flexGrow: 1,
+  },
+  footer: {
+    backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(6),
+  },
+  actions:{
+    margin:'auto'
+  },
+  white: {
+    color: "#fff",
+    textShadow: '2px 2px 4px #000000'
+  }
+}));
+
+
+const cards1 = [1, 2, 3];
+
+export default function Home() {
+  const classes = useStyles();
+  
+  const [productsFeature, setProductsFeature] = useState([]);
+
+  useEffect(() => {
+    // GET request using axios inside useEffect React hook
+    axios.get('http://localhost:8080/api/products?label=feature')
```

Comment:
Base URL (http://localhost) and PORT (8080) should be in env variables, and it would be great if you had ApiService which has methods (with url, query params.. parameters) that handle all get, post etc axios requests